rubygem-foreman_remote_execution_core/rubygem-foreman_remote_execution_core.spec

```diff
@@ -0,0 +1,61 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name foreman_remote_execution_core
+
+Summary: Foreman remote execution - core bits
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 1.0.1
+Release: 1%{?foremandist}%{?dist}
+Group: Development/Libraries
+License: GPLv3
+URL: https://github.com/theforeman/foreman_remote_execution
+Source0: http://rubygems.org/downloads/%{gem_name}-%{version}.gem
+
+Requires: %{?scl_prefix}rubygem(net-scp)
+Requires: %{?scl_prefix}rubygem(net-ssh)
+Requires: %{?scl_prefix}rubygem(foreman_tasks_core)
```

Comment:
1. This must be `foreman-tasks-core`, it's the name of the gem and what the other package provides.
1. Version specification should be `>= 0.1.0` and `< 0.2.0` to match the gemspec.